sources/engine/Stride.Graphics/GraphicsResourceBase.cs

```diff
@@ -90,6 +98,7 @@ protected internal virtual void OnResume()
         /// <inheritdoc/>
         protected override void Destroy()
         {
+            GC.SuppressFinalize(this);
```

Comment:
Is this ok to call `SuppressFinalize` in here, when called from the finalizer?

Reply:
Yes, I tested it a bunch and [microsoft's docs](https://docs.microsoft.com/en-us/dotnet/api/system.gc.suppressfinalize?view=net-6.0) report that:
> If obj does not have a finalizer or the GC has already signaled the finalizer thread to run the finalizer, the call to the SuppressFinalize method has no effect.